### DISCUSSION TITLE
xmrwallet.com/.org: badware

### DISCUSSION THREAD

**Discussion by HardenedSteel** [uBlockOrigin/uAssets#25202] [General]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://xmrwallet.com/
https://xmrwallet.org/
```

### Description

Claims to be open source but its not, github repos are empty, known to deleting github issues/comments.

https://web.archive.org/web/20201123203247/https://github.com/XMRWallet/Website/issues/11
https://web.archive.org/web/20201123203247/https://github.com/XMRWallet/Website/issues/12
https://web.archive.org/web/20201123203246/https://github.com/XMRWallet/Website/issues/13
https://web.archive.org/web/20201123203204/https://github.com/XMRWallet/Website/issues/15

users also report its scam on Reddit: https://www.reddit.com/r/Monero/search/?q=xmrwallet.com&type=link

### Other extensions used

n/a

### Screenshot(s)

n/a

### Configuration

n/a

**Comment by stephenhawk8054** [comment]
We don't actually address scam category in badware list: https://github.com/uBlockOrigin/uAssets/discussions/24317#discussioncomment-9930759

**Comment by HardenedSteel** [comment]
does backdoor fall under scam or badware?

**Comment by stephenhawk8054** [comment]
It depends. But in any case, we need credible sources of reports / analysis before adding to the list.

**Comment by HardenedSteel** [comment]
private keys generated server-side and source code isn't available while it says open source.

**Comment by stephenhawk8054** [comment]
I'll convert to discussion
